specs/pvs/corla.pvs

```diff
@@ -2056,9 +2055,15 @@ BEGIN
     [# location: string, date: string #]
   rla_information_table: TYPE =
     sequence[[RLA, audit_reason, contest, risk_limit]]
-% @todo nealmcb rename to audited_ballot_list_table?
-  ballots_to_audit_list_table: TYPE =
+%%% We formalize the list of ballots under audit and organize them
+%%% both by contest and by county. The former is defined when the RLA
+%%% algorithm randomly chooses ballots. The latter is populated by
+%%% filtering the former by county, and it is what is shared with
+%%% counties via the Audit Board Dashboard.
+  audited_ballot_by_contest_list_table: TYPE =
     sequence[[contest, sequence[ballot_manifest_info]]]
```

Comment:
Per discussion with @dmz, we don't choose by contest. We choose by county and stratify by county across contests.
We will probably end up filtering by contest from that. But that is very tricky to do since we don't trust the CVRs. So we need independent-of-the-voting-system ballot manifest evidence of counts by contest, which we don't have yet.
This affects multi-sheet ballots since they are provided as one CVR per ballot sheet it seems.